jobs/JGDAS_ENKF_SELECT_OBS

```diff
@@ -102,20 +102,20 @@ status=$?
 
 # Link observational data
 export PREPQC="${COMIN_OBS}/${OPREFIX}prepbufr"
-if [ ! -f ${PREPQC} ]; then
+if [[ ! -f ${PREPQC} ]]; then
     echo "WARNING: Global PREPBUFR FILE ${PREPQC} MISSING"
 fi
 export TCVITL="${COMIN_ANL}/${OPREFIX}syndata.tcvitals.tm00"
 if [[ ${DONST} = "YES" ]]; then
    if [[ ${MAKE_NSSTBUFR} == "YES" ]]; then
-      export NSSTBF="${COMOUT}/${OPREFIX}nsstbufr"
+      export NSSTBF="${COMIN_OBS}/${OPREFIX}nsstbufr"
    else
       export NSSTBF="${COMIN_OBS}/${OPREFIX}nsstbufr"
    fi
 fi
 
 if [[ ${MAKE_ACFTBUFR} == "YES" ]]; then
-   export PREPQCPF="${COMOUT}/${OPREFIX}prepbufr.acft_profiles"
+   export PREPQCPF="${COMIN_OBS}/${OPREFIX}prepbufr.acft_profiles"
```

Comment:
Same as above

Reply:
Same as above

Reply:
Please eliminate the if-block and simply retain the `export PREPQCPF` line.

Reply:
Do you want me to remove this line 113?

Reply:
There is no else statement in this if logic in line 113.